edk2toollib/windows/capsule/inf_generator.py

```diff
@@ -14,6 +14,15 @@
 import uuid
 
 
+class InfSection(object):
+    def __init__(self, name) -> None:
+        self.Name = name
+        self.Items = []
+
+    def __str__(self) -> str:
```

Comment:
can you put a trivial unit test in place for this?

Reply:
Done.